### PR TITLE
use the latest `selenium/standalone-chrome`

### PR DESCRIPTION
**Pull request by no-reply**
selenium has changed its terminology to `allowed-ips`. let's take advantage of
that, and upgrade besides.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ services:
       - rails-tmp:/app/samvera/hyrax-webapp/tmp
 
   chrome:
-    image: selenium/standalone-chrome:3.141
```
Comment (dunn) [inline docker-compose.yml:38]: this version scheme is still in use, and we might want to keep using it to make it easier to match the version on the ruby end: https://hub.docker.com/r/selenium/standalone-chrome/tags?page=1&ordering=last_updated

Comment (no-reply) [reply docker-compose.yml:38]: good call. in the meanwhile, i'm able to confirm that i could run the feature specs in docker-compose with this PR.

**Review by dunn — APPROVED** [review]

**Comment by no-reply** [comment]
well, with the rolled back version, no luck. are we sure we want `3.141`.

the ruby gem is at `3.142.7`, and chromium itself is on `90`. i think we might just want 90.

**Review by jeremyf — APPROVED** [review]

**Comment by no-reply** [comment]
@jeremyf were you able to get this working?

my last attempt at end of day yesterday was broken on my end, per: https://github.com/samvera/hyrax/pull/4923#issuecomment-833173148

**Pull request merged**